content/en/docs/guides/namespace-migration-across-tenants.md

```diff
@@ -28,7 +28,7 @@ ns-bar   Active   28s   capsule.clastix.io/tenant=wind,kubernetes.io/metadata.na
 ns-foo   Active   36s   capsule.clastix.io/tenant=solar,kubernetes.io/metadata.name=ns-foo
 0df8e9ee-5f6f-40a4-897d-b80d349ca36f%
 ```
-Edit UID and labels to match the output of the previous commands.
+While altering ownerReferences name is sufficient on its own, it's highly recommended to edit the UID and labels to match the output of the previous commands.
```

Comment:
> it's highly recommended to edit the UID and `labels`
Looks good, just one thing, we only need to change tenant name and tenant UID, not the labels .
If you change the label and save, you will find the label stay same, namespace still belong to the old tenant. And if you change the tenant name and UID, the label will be automatically changed.

Reply:
Fixed and also adjusted the wording in the previous paragraphs to match the findings.

Reply:
Lovely 👍 , thanks.